backend/utils/graph_db_utils_test.py

```diff
@@ -0,0 +1,64 @@
+from unittest.mock import MagicMock
+from neo4j import Driver, Session
+import pytest
+# We assign an alias to "test_connection" to avoid pytest treating it as another test function
+from utils.graph_db_utils import test_connection as verify_connection, create_goal
```

Comment:
Is there a reason we're changing the `test_connection` name to `verify_connection`? If we think it should be renamed I'd prefer we rename it in `graph_db_utils.py` rather than adjust it's name just here

Reply:
Interesting, that is a funny issue with `pytest`. With that context I'm happy with either the alias or calling it `verify_connection`